src/api_server/src/lib.rs

```diff
@@ -425,7 +418,7 @@ mod tests {
         assert_eq!(response.status(), StatusCode::NoContent);
 
         let response = api_server.put_mmds(serde_json::Value::Bool(true));
-        assert_eq!(response.status(), StatusCode::BadRequest);
+        assert_eq!(response.status(), StatusCode::NoContent);
```

Comment:
Any reason why we want to keep this check? It was previously there to check the error case. If we want to check that all values are valid values, then we should add a test for all. Otherwise, I would remove this one because it is confusing when reading the test.

The same applies to the following assert.